### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by QuinnWass**
Created TabGraph module for use in background

### CODE REVIEW

File: modules/TabGraph.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    AddTopLevelTab(tabNode)
+    Used to add a top level tab into the TabGraph. Top level tabs are created by the browser independent of
+    current tabs. TabNodes must be constructed properly before insert
+    */
+    AddTopLevelTab(tabNode){
+        let tabId = tabNode._id
+        if(!this.CheckTabIdExists(tabNode)){
+            throw "No id for TabNode"
+        }
+        this._topLevelTabs.append(tabid)
```
Comment (WrenChris) [inline modules/TabGraph.js]: capitalization... JavaScript is case sensitive, correct?

Comment (QuinnWass) [reply modules/TabGraph.js]: Are you referring to line 51? I will fix and recommit

Comment (QuinnWass) [reply modules/TabGraph.js]: fixed

Comment (WrenChris) [reply modules/TabGraph.js]: yes, thank you!

**Pull request merged**